lang/clang-11-bootstrap/Portfile

```diff
@@ -342,7 +342,7 @@ post-patch {
     }
 }
 
-if {${universal_possible} && [variant_isset universal]} {
+if {${universal_possible} && [variant_isset universal] && [llength ${configure.universal_archs}] > 2} {
```

Comment:
you want this block to run only if the build is targeting 3 universal archs or more?

what systems would that be?

also I guess all the other universal builds with 2 archs that succeeded will have to be revbumped to rebuild with this new constraint.

Maybe I don't follow exactly how this works -- that is quite possible.

Reply:
@kencu nope, you have catched a typo. Thanks for it!